typethrift-all

```diff
@@ -0,0 +1,15 @@
+set -e
```

Comment:
This is a "build" script that will run the node generator on an arbitrary IDL file and place the generated code. Not sure why it was removed upstream, I've found it useful for getting off the ground.